app/main.py

```diff
@@ -1 +1,26 @@
-# write your code here
+
+def copy_file(command: str) -> str | None:
+    new_str = command.split()
+    if len(new_str) < 3:
+        return "argument is not correct"
+    file_first = new_str[1]
+    file_second = new_str[2]
+    cmd = new_str[0]
+    if cmd != "cp":
+        return "Command is not correct"
+
+    if file_first == file_second:
```

Comment:
All above code can be simplified to one or two if statements. So remove unnecessary return and pass instructions. Try to check if len is correct, if command is correct and if file name is not the same and then put rest of the code under this condition if it is satisfied, if it's not satisfied - don't do anything.  Also maybe you can use unpacking instead of accessing elements by index